### PR TITLE
Transition to monorepo

### PR DESCRIPTION
**Pull request by alexanbj**
This pull request transitions the repo to a [monorepo](https://github.com/babel/babel/blob/master/doc/design/monorepo.md). [Bolt](https://github.com/boltpkg/bolt) is used to manage the monorepo.

Currently the Dockerfile for the public facing frontend is moved to the top of the repo, so we probably need to do something about this for the admin frontend once we get started on that.

This resolves GlobalDigitalLibraryio/issues#358

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,28 +3,13 @@ FROM node:8.11.1-alpine
...
-COPY server $APP_PATH/server
-COPY hocs $APP_PATH/hocs
-COPY style $APP_PATH/style
-COPY locale $APP_PATH/locale
-COPY lib $APP_PATH/lib
-COPY routes.js $APP_PATH
-COPY .babelrc $APP_PATH
-COPY config.js $APP_PATH
-COPY fetch.js $APP_PATH
-COPY types.js $APP_PATH
```
Comment (runarfu) [inline Dockerfile]: Skulle disse fjernes? Blir det med nå?

File: README.md [hunk 2]
```diff
@@ -1,34 +1,26 @@
...
+# Installation
 
-Start node server with hot reloading middleware listening on port 3000.
+#### Before you start
 
-```
-$ yarn run dev
-```
+* [node](https://nodejs.org/) version should be 8 or above (to check `node -v`) or use [nvm](https://github.com/creationix/nvm).
+* [yarn](https://yarnpkg.com/) should be installed globally.
```
Comment (runarfu) [inline README.md]: Legge til punkt her om å installere bolt (istedet for å bare nevne det under)?

File: README.md [hunk 3]
```diff
@@ -1,34 +1,26 @@
...
 
-### Translations
-For i18n we use [lingui](https://github.com/lingui/js-lingui).
+#### Install
 
-#### Updating translations
+Each module lives in it's own package under the `packages` directory. We are using [bolt](https://github.com/boltpkg/bolt) to manage our monorepo, so you are going to need bolt installed globally as well.
```
Comment (runarfu) [inline README.md]: its

File: packages/gdl-frontend/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+$ yarn run dev
+```
+
+### Translations
+
+For i18n we use [lingui](https://github.com/lingui/js-lingui).
+
+#### Updating translations
+
+Whenever new that are to be translated using lingui are added, run the following command to update the existing translation files:
```
Comment (runarfu) [inline packages/gdl-frontend/README.md]: new …?

File: Dockerfile (hunk 1 above)
Comment (alexanbj) [reply Dockerfile]: Ja, for nå kopierer vi hele gdl-frontend mappa

**Pull request merged**